config/database.yml

```diff
@@ -5,7 +5,7 @@ default: &default
   min_messages: warning
   pool: <%= Integer(ENV.fetch("DB_POOL", 5)) %>
   reaping_frequency: <%= Integer(ENV.fetch("DB_REAPING_FREQUENCY", 10)) %>
-  timeout: 5000
+  timeout: 120000
```

Comment:
I believe that this timeout is for how long you wait for an available connection from the pool before raising an error.  This has nothing to do with how long you are using the connection for.  Do you have some other indication that this is the trouble?

Reply:
Ah, I was using this in debugging with no result. so closing.